### PR TITLE
Conditionals

### PR DESCRIPTION
**Pull request by VisnetAppel**
### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/c-sharp/concepts/Conditionals/conditionals.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
+---
+Title: 'conditionals'
+Description: 'Compares inputs and gives an boolean output depending on it being True or False.'
+Subjects:
```
Comment (yangc95) [inline content/c-sharp/concepts/Conditionals/conditionals.md:4]: Please double-check these with the [subjects](https://github.com/Codecademy/docs/blob/main/documentation/subjects.md) document.

File: content/c-sharp/concepts/Conditionals/conditionals.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+  - 'Computer Science'
+  - 'Web Dev'
+  - 'App Dev'
+  - 'Game Dev'
+Tags:
```
Comment (yangc95) [inline content/c-sharp/concepts/Conditionals/conditionals.md:9]: Please see the [tags](https://github.com/Codecademy/docs/blob/main/documentation/tags.md) documentation for tags that you can use.

File: content/c-sharp/concepts/Conditionals/conditionals.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+  -
+CatalogContent:
```
Comment (yangc95) [inline content/c-sharp/concepts/Conditionals/conditionals.md:11]: Please refer to the [catalog content](https://github.com/Codecademy/docs/blob/main/documentation/catalog-content.md) document.

File: content/c-sharp/concepts/Conditionals/conditionals.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+  - 'Learn C#'
+---
+
+## Explanation
```
Comment (yangc95) [inline content/c-sharp/concepts/Conditionals/conditionals.md:15]: Refer to the [concept template](https://github.com/Codecademy/docs/blob/main/documentation/entry-template.md) for help with structure

File: content/c-sharp/concepts/Conditionals/conditionals.md [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+
+```cs
+if(output === true){
+    Console.WriteLine('I returned true')
+}
+
+```
+>**note** `===` means equal. `!==` means not equal. 
+
+## Explanation of example 1
```
Comment (yangc95) [inline content/c-sharp/concepts/Conditionals/conditionals.md:45]: These explanations don't require their own header. I suggest adding the explanation right underneath the code block. It's best to make the explanation simply and straightforwardly.

File: content/c-sharp/concepts/Conditionals/conditionals.md [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+
+## Example 1
+```cs
+
+var input1 = 10;
+var input2 = 10;
+var output;
+
+if (input1 === input2){
+    output = true; // sets the value of output to true.
```
Comment (yangc95) [inline content/c-sharp/concepts/Conditionals/conditionals.md:31]: Capitalize comments (here and all below)

File: content/c-sharp/concepts/Conditionals/conditionals.md [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+
+We start by creating three variables (`var`) and assigning a value to two of them. The third variable will be assigned in a later stage of the code.
+
+After creating the variables we start our `if statement`.
+
+`if` input1 is equal to input 2 then set the variable of output to true. In our example, input1 was equal to input 2. The `if statement` check was true so it ran the code setting the value of output to true. 
+
+after that, we wrote a second `if statement` which checked if the output was equal to true. The output of this was true so it ran the code and wrote *I returned true* to the console. if the first `if statement` would have been false. The output would not get assigned a value and the second `if statement` would return false meaning no text will be written to the console.
+
+## else statements
```
Comment (yangc95) [inline content/c-sharp/concepts/Conditionals/conditionals.md:55]: Headings with mention of code should also be placed in backticks

File: content/c-sharp/concepts/Conditionals/conditionals.md [hunk 8]
```diff
@@ -0,0 +1,152 @@
...
+
+A `else statement` goes in combination with the `if statement` and returns a value `if` false.
+
+## Example 2
+```cs
```
Comment (yangc95) [inline content/c-sharp/concepts/Conditionals/conditionals.md:60]: Made sure to add your 2-space indents to all code blocks!

File: content/c-sharp/concepts/Conditionals/conditionals.md [hunk 9]
```diff
@@ -0,0 +1,152 @@
...
+
+`if` input1 is equal to input 2 then set the variable of output to true. In our example, input1 was equal to input 2. The `if statement` check was true so it ran the code setting the value of output to true. 
+
+after that, we wrote a second `if statement` which checked if the output was equal to true. The output of this was true so it ran the code and wrote *I returned true* to the console. if the first `if statement` would have been false. The output would not get assigned a value and the second `if statement` would return false meaning no text will be written to the console.
+
+## else statements
+
+A `else statement` goes in combination with the `if statement` and returns a value `if` false.
+
+## Example 2
```
Comment (yangc95) [inline content/c-sharp/concepts/Conditionals/conditionals.md:59]: There should be a separation between different elements in the entry (headings, code blocks, notes, etc.)

File: content/c-sharp/concepts/Conditionals/conditionals.md [hunk 10]
```diff
@@ -0,0 +1,152 @@
...
+```cs
+
+string getInput1(input1) => input1 === 10 ? "I returned true" : "I returned false";
+
+Console.WriteLine(getInput1(10)); // Prints to the console "I returned true"
+Console.WriteLine(getInput1(5)); // Prints to the console "I returned false"
+
+```
+
+## Explanation of example 4
```
Comment (yangc95) [inline content/c-sharp/concepts/Conditionals/conditionals.md:148]: I see 4 example, and while I do applaud the number of examples, I'd like to see either more meaning headings that show what conditional concept it's covering.

**Review by yangc95 — COMMENTED** [review]
@VisnetAppel A quite lengthy entry! I left some comments that I would like to be addressed. After my approval we can move on to another review. Appreciate your work on this 🙂

**Comment by SSwiniarski** [comment]
@VisnetAppel, it's been over two weeks, do you need assistance with implementing @yangc95 edits?

Also, you need to add the email used for this entry to your GitHub account so the CLA knows who you are.

**Comment by VisnetAppel** [comment]
Hello,

I don't think I need help just yet. I've not had a lot of free time lately.
I"ll try to give it another update this weekend.

Thanks for checking up on me

Op wo 22 mrt 2023 om 17:31 schreef SSwiniarski ***@***.***>:

**Comment by VisnetAppel** [comment]
I've accidently closed it on my side I believe. @yangc95 i've applied your feedback and started a new pull reqeust (1916).

**Pull request closed without merging**